compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -686,8 +686,8 @@ error.iterable.not.supported.collection=\
   incompatible types: ''{0}'' is not an iterable collection
 
 error.incompatible.iterator.function.signature=\
-  iterable objects must have a __iterator function with signature, ' \
-  public function __iterator() returns (object { public function next () returns (record {| T value; |}?); })';
+  iterable objects must have an iterator method with signature, ' \
+  public function iterator() returns (object { public function next () returns (record {| T value; |}?); })';
```

Comment:
Do we have tests for this?

Reply:
Removed this as being distinct function signature is anyway tested.